### PR TITLE
feat: toast

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: components/toast/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// tostbar2
+function leadingToastFunc() {
+    let snackbarTwo = document.getElementById("leading-toast");
+    snackbarTwo.className = "show";
+    setTimeout(function () {
+        snackbarTwo.className = snackbarTwo.className.replace("show", "");
+    }, 3000);
+}
+
+// function toastBarFuncThird() {
```
Comment (prajapati-priyanka) [inline components/toast/index.js:18]: remove this commented code

File: components/toast/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+//     let snackbarThree = document.getElementById("stack-toast");
+//     snackbarThree.className = "show";
+//     setTimeout(function () {
+//         snackbarThree.className = snackbarThree.className.replace("show", "");
+//     }, 3000);
+// }
```
Comment (prajapati-priyanka) [inline components/toast/index.js:24]: remove these commented code

File: components/toast/toast.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                        <li><a href="/components/button/button.html">Button</a></li>
+                        <li><a href="/components/card/card.html">Card</a></li>
+                        <li><a href="/components/image/image.html">Image</a></li>
+                        <li><a href="/components/input/input.html">Input</a></li>
+                        <li><a href="/components/typography/typography.html">Typography</a></li>
+                        <li><a href="/components/list/list.html">List</a></li>
+                        <li><a href="/components/navigation/navigation.html">Navigation</a></li>
+                        <li><a href="/components/modal/modal.html">Modal</a></li>
+                        <li><a href="/components/rating/rating.html">Rating</a></li>
+                        <li><a href="/components/toast/toast.html">Tost</a></li>
```
Comment (prajapati-priyanka) [inline components/toast/toast.html]: correct toast spelling

File: components/toast/toast.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                            ------------------------------------------------------------
+
+                            // tostbar2
+                            function leadingToastFunc() {
+                                let snackbarTwo = document.getElementById("leading-toast");
+                                snackbarTwo.className = "show";
+                                setTimeout(function () {
+                                    snackbarTwo.className = snackbarTwo.className.replace("show", "");
+                                }, 3000);
+                            }
```
Comment (prajapati-priyanka) [inline components/toast/toast.html:132]: Here also, remove commented code

**Review by prajapati-priyanka — APPROVED** [review]
Just see the suggested changes. Rest everything looks fine. Toast functionality also working fine.
Great work, @vermasavita

File: components/toast/index.js (hunk 2 above)
Comment (vermasavita) [reply components/toast/index.js:24]: This commented code would be needed later

File: components/toast/toast.html (hunk 3 above)
Comment (vermasavita) [reply components/toast/toast.html]: Opps yes. Corrected

**Review by Husain01 — APPROVED** [review]
Looks good to go

**Pull request merged**